src/WebApp/wwwroot/js/site.js

```diff
@@ -22,4 +22,32 @@ for (let i = 0; i < headerSlideTriggers.length; i++) {
         // Prevent navigation
         e.preventDefault();
     });
-}
\ No newline at end of file
+}
+
+/*
+ * Date/Time functions
+ ----------------
+ * Utilities to handle dates
+*/
+function utcDateTimeToLocalDisplay(date) {
```

Comment:
Have we got Babel preprocessing set up yet? If not, we should make that a priority. If we do, this file should be reviewed and updated to ES6 - notably, `const` and `() => { }`.

Reply:
Not yet basically. Keeping it ES3 so far

Reply:
Works for now. Just bear in mind that the longer it takes to get it set up, the more JS there'll be to rewrite when it's ready :)